src/renderers/testing/ReactTestRenderer.js

```diff
@@ -62,9 +63,13 @@ ReactTestComponent.prototype.receiveComponent = function(
 };
 ReactTestComponent.prototype.getHostNode = function() {};
 ReactTestComponent.prototype.getPublicInstance = function() {
-  // I can't say this makes a ton of sense but it seems better than throwing.
-  // Maybe we'll revise later if someone has a good use case.
-  return null;
+  if (this._mockConfig) {
+    var { getMockRef } = this._mockConfig;
+    if (getMockRef) {
+      return getMockRef(this._currentElement);
```

Comment:
Can we make a default `mockConfig` implementation and always assume `mockConfig` exists?